### PR TITLE
[UI] Adapter migration to MUI v5

### PR DESCRIPTION
**Pull request by SAHU-01**
**Notes for Reviewers**

- This PR migrates adapter page components to MUI v5 using sistent theme and styling

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — DISMISSED** [review]
✅

**Comment by sudhanshutech** [comment]
@SAHU-01 eslite issues....

File: ui/components/MesheryPlayComponent.js [hunk 1]
```diff
@@ -241,51 +198,53 @@ const MesheryPlayComponent = (props) => {
...
 
-const mapDispatchToProps = (dispatch) => ({
-  setAdapter: bindActionCreators(setAdapter, dispatch),
-});
-
+// Mapping state to props
 const mapStateToProps = (state) => {
   const k8sconfig = state.get('k8sConfig');
-  const meshAdapters = state.get('meshAdapters');
+  const meshAdapters = state.get('meshAdapters').toJS(); // Assuming meshAdapters is Immutable
```
Comment (sudhanshutech) [inline ui/components/MesheryPlayComponent.js:239]: why this change?

**Review by leecalcote — APPROVED** [review]
All tests passing.
